app/main/views/frameworks.py

```diff
@@ -1353,27 +1353,38 @@ def sign_framework_agreement(framework_slug):
 
     supplier = data_api_client.get_supplier(current_user.supplier_id)["suppliers"]
     company_details = get_company_details_from_supplier(supplier)
-    form = SignFrameworkAgreementForm()
+    declaration = data_api_client.get_supplier_declaration(current_user.supplier_id, framework_slug).get('declaration')
     framework_pdf_url = content_loader.get_message(framework_slug, 'urls').get('framework_agreement_pdf_url')
-    contract_titles = {
-        'g-cloud-12': 'Framework Agreement'
+    framework_specific_labels = {
+        'g-cloud-12': {'title': 'Sign agreement',
+                       'contract_title': 'Framework Agreement',
+                       'include_govuk_link': False},
+        'digital-outcomes-and-specialists-5': {'title': 'Sign contract',
+                                               'contract_title': 'Framework Award Form',
+                                               'include_govuk_link': True}
```

Comment:
Have we documented that this will need to be updated for G13/DOS6?

Reply:
As discussed elsewhere it is likely this process will change again in future, but it may make sense to add some pointers to the manual about updating e-signature content.